<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SavesMenu</class>
 <widget class="QWidget" name="SavesMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>425</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Меню Сохранений</string>
  </property>
  <widget class="QLabel" name="Main_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Farmer's Valley</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveC_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3 Слот</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveA_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1 Слот</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Exit_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Вернуться</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveD_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4 Слот</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveB_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2 Слот</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
